### ISSUE TITLE
groups similar methods/classes

### ISSUE THREAD

**Issue by erickvermot** [kytos/python-openflow#315]
- Group similarities as in HWAdress and DPID.
The have very similar pack functions for example.
Maybe should they be derived from a same base class ("colon separated hex
values...")
Same with echo request, echo reply, and hello... and many others..

Maybe implement using Mixin classes, as proposed by Diego. I agree;

**Comment by beraldoleal** [comment]
I Agree. Definitely this need a refactoring. There is a few duplicated code there. 

For instance `_fmt` is not being used in some cases. But lets discuss this on future refactoring.
